Gemfile

```diff
@@ -13,16 +13,29 @@ gem 'test-unit', '~> 3.0' if rails_version[0] == '3'
 
 if rails_version == 'master'
   gem 'arel',       github: 'rails/arel'
-  gem 'sprockets',  github: 'rails/sprockets'
-  gem 'sass-rails', github: 'rails/sass-rails'
+  # gem 'sprockets',  github: 'rails/sprockets'
+  # gem 'sass-rails', github: 'rails/sass-rails'
   gem 'rack',       github: 'rack/rack'
-  gem 'sprockets-rails', '3.0.0.beta2'
+  gem 'devise',     github: 'plataformatec/devise'
+  gem 'ransack',    github: 'activerecord-hackery/ransack'
+  gem 'kaminari',   github: 'amatsuda/kaminari', branch: '0-17-stable'
+  gem 'draper',     github: 'audionerd/draper', branch: 'rails5', ref: 'e816e0e587'
+  gem 'formtastic', github: 'justinfrench/formtastic'
+  gem 'activemodel-serializers-xml', github: 'rails/activemodel-serializers-xml' # drapergem/draper#697
+  %w[rspec-core rspec-expectations rspec-mocks rspec-rails rspec-support].each do |lib|
+    gem lib, github: "rspec/#{lib}"
+  end
+  gem 'rack-mini-profiler', github: 'MiniProfiler/rack-mini-profiler'
+  gem 'database_cleaner',  github: 'pschambacher/database_cleaner', branch: 'rails5.0', ref: '8dd9fa4'
+  gem 'activerecord-jdbc-adapter', github: 'jruby/activerecord-jdbc-adapter', platforms: :jruby
 end
 
 # Optional dependencies
 gem 'cancan'
-gem 'devise'
-gem 'draper'
+if rails_version != 'master'
+  gem 'devise'
+  gem 'draper'
+end
```

Comment:
The problem is Bundler raises an error if you define a gem multiple times in your Gemfile.

I tried to keep the Github gem requirements in a single section of the file, so that once Rails is released that section can be deleted, and then we can remove the `if rails_version != 'master'` conditionals in the rest of the file.